src/components/Chat.js

```diff
@@ -89,9 +89,11 @@ class TextInput extends React.Component {
 
 
 const StyledTextComposer = styled.div`
-  padding:0.5em;
   background:#fff;
   border-top:1px solid rgba(0,0,0,0.1);
+  display: flex;
+  padding-top: 5px;
+  padding-left: 5px;
```

Comment:
missing padding bottom (I'm assuming you only wanted to remove the right padding where the Send button is)

Reply:
Also, what if there's no button?